configs/cdappconfig.json

```diff
@@ -0,0 +1,63 @@
+{
+  "database": {
+    "adminPassword": "postgres",
+    "adminUsername": "postgres",
+    "hostname": "localhost",
+    "name": "idmsvc",
+    "password": "idmsvc",
+    "port": 5432,
+    "sslMode": "disable",
+    "username": "idmsvc"
```

Comment:
Could you please use different values for name, user, and password like in `deployments/docker-compose.yaml` ?

Reply:
Thanks! updated